tests/test_api/test_coverage_learning_endpoints.py

```
@@ -0,0 +1,1171 @@
+        accuracy = resp.json()["llm_accuracy"]
+        assert accuracy["engagement_rate"] is None
+        assert accuracy["reversal_rate"] is None
+
+
+# ---------------------------------------------------------------------------
+# GET /health
+# ---------------------------------------------------------------------------
+
+class TestGetLearningHealth:
+    def test_health_ok_when_fb_returns_ok(self, learning_client):
+        client, _, mock_fb, _ = learning_client
```

Ruff (0.15.14)

[warning] Unpacked variable `client` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)